ledger/sandbox-common/src/test/lib/scala/platform/sandbox/services/TestCommands.scala

```diff
@@ -31,6 +41,27 @@ trait TestCommands {
 
   protected def templateIds = new TestTemplateIdentifiers(packageId)
 
+  def uploadPackageFiles(
```

Comment:
Usage of this method within each test case and sometimes within wrapper around ledger api calls changes the dynamic behavior of the tests. Now each ledger api call is preceded by a package upload. It is a time consuming and synchronous operation. I would expect the overall CI execution time to go up.
I'd much rather, this operation were called in a "before" block preferably in a separate trait, so that it is done only once per each suite.
I would then also move this method out of this fixture where it feels slightly out of place. Please note that otherwise all other functions here are pure.